.changeset/four-lamps-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Disallow status codes outside 400-599
```

Comment:
```suggestion
[breaking] Disallow status codes outside 400-599
```

Do we want to label this as breaking? `throw error(204)` _was_ being used by one of our own codebases, so I don't think it's unreasonable to assume that someone else would have done it too.